src/main/java/desktop/pages/SignInPage.java

```diff
@@ -0,0 +1,49 @@
+package desktop.pages;
+
+import abstractclasses.page.AbstractPage;
+import desktop.fragments.NavigationBar;
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static constants.Constants.SIGN_IN_PAGE_URL;
+import static java.lang.String.format;
+
+@Getter
+public class SignInPage extends AbstractPage {
+    @FindBy(xpath = "//form[@id='ap_register_form']//input[@type='text']")
+    private WebElement nameInputField;
+    @FindBy(xpath = "//form[@id='ap_register_form']//input[@type='email']")
```

Comment:
please add empty lines between WebElements to improve code readability:
<img width="905" alt="image" src="https://user-images.githubusercontent.com/90613721/166507657-7cc1e953-a76f-444d-9122-91fdf1ff60d2.png">